src/main/views/MattermostView.ts

```diff
@@ -335,10 +335,6 @@ export class MattermostView extends EventEmitter {
     }
 
     updateMentionsFromTitle = (title: string) => {
-        if (this.serverInfo.remoteInfo.serverVersion && Util.isVersionGreaterThanOrEqualTo(this.serverInfo.remoteInfo.serverVersion, '5.29.0')) {
```

Comment:
Apparently I didn't read and thought that this function only needed to happen on v5.28 and below 😆

Reply:
only for the asterisk, at some point we should be able to remove that code as we don't expect people to run very old versions of the server.